lib/vsc/utils/nagios.py

```diff
@@ -434,6 +438,11 @@ class SimpleNagios(NagiosResult):
 
     def __init__(self, **kwargs):
         """Initialise message and perfdata"""
+        self._init(**kwargs)
+
+    def _init(self, reporterclass=NagiosReporter, **kwargs):
```

Comment:
why not use a class constant instead of passing the extra arg?

Reply:
that's indeed a simpler solution
fixed in https://github.com/hpcugent/vsc-utils/pull/81/commits/507804c29499c399c97d2809de214eb64f778473